src/layouts/MainLayout.vue

```diff
@@ -41,7 +90,8 @@
             <q-item-label header v-if="item.title" :key="item.title">{{item.title}}</q-item-label>
             <q-item v-for="link of item.items"
                     :key="link.text"
-                    clickable v-ripple
+                    clickable
+                    :disable="isPhantom && link.disable" v-ripple
```

Comment:
There is no visual feedback to the user regarding why those buttons are disabled, could be misleading.